desc/compute/_equil.py

```diff
@@ -677,6 +681,7 @@ def _W_B(params, transforms, profiles, data, **kwargs):
     data=["B", "sqrt(g)"],
 )
 def _W_Bpol(params, transforms, profiles, data, **kwargs):
+    errorif(kwargs.get("basis", "rpz").lower() != "rpz", NotImplementedError)
```

Comment:
why does this matter here? is it because some of the intermediate quantities get messed up when basis is xyz, so B itself is wrong at this point?

Reply:
This compute function assumes the basis of its dependencies are in cylindrical coordinates.

Reply:
Again, #1027 solves this.